File: HippocratesWebApp/Vakcina.php
<?php
class Vakcina{
    public $ime;
    public $sifra;
    public $opis;
    
    public function __construct($sifra,$ime,$opis){
        $this->ime=$ime;
        $this->sifra=$sifra;
        $this->opis=$opis;
    }
}
?>

File: Hippocrates/Hippocrates.Data/Entiteti/Vakcina.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hippocrates.Data.Entiteti
{
    public class Vakcina
    {
        public virtual string Ime { get; set; }
        public virtual string Sifra { get; set; }
        public virtual string Opis { get; set; }

        public virtual IList<PrimioVakcinu> PrimioVakcinuPacijenti { get; set; }

        public Vakcina()
        {
            PrimioVakcinuPacijenti = new List<PrimioVakcinu>();
        }
    }
}
